core/lib/presentation/utils/html_transformer/html_template.dart

```diff
@@ -30,6 +30,7 @@ String generateHtml(String content, {
   String? resize,
   String? cursor,
 }) {
+  content = content.replaceAll('&quot;', '');
```

Comment:
WDYT @dab246 ? any side effect?

Reply:
I don't know what your intention is when replacing that character. IMO it will remove the apostrophe " characters in the email body. You should double check it.

Reply:
please more detail on the case we need to check.

Reply:
As Manh explain that will remove the redundant `&quot;` for the base 64, ....bla bla bla

Reply:
Please provide details about this check. I don't think it's like that?

Reply:
@dab246  When using function getText of enough_html_editor, `\"` will replace to `&quot;`. So when you send body content to the server, it will be wrong. So I remove it for that. I change this code to the below. Please check this.
https://github.com/linagora/tmail-flutter/pull/757/files#diff-dfc6275288275045d8a26d4b6c04d7b180f0ccd025f6f1564400a14b744c5d18R142

Reply:
here is image code of package
<img width="635" alt="Screen Shot 2022-07-26 at 17 12 10" src="https://user-images.githubusercontent.com/107173849/180984102-ded6ca01-599d-47d2-995c-f4f3c3a4f31f.png">

Reply:
So why do you replace it with a blank character? `content = content.replaceAll('&quot;', '');`. If you do so, double-quoted email messages will not be displayed properly.

Reply:
It can affect:
- `saveDraft` when the two content before and after composing email are different.
- The content when `reply`, `forward` email is incorrect

Reply:
@dab246 You are right. I changed it please review it

Reply:
> It can affect:
> 
> * `saveDraft` when the two content before and after composing email are different.
> * The content when `reply`, `forward` email is incorrect

Yeb I fixed it